### PR TITLE
Bootstrap 3 Migration: Messaging > Case Groups

### PR DESCRIPTION
**Pull request by pnaw94**
@biyeun 
scr01
scr02

### CODE REVIEW

File: corehq/apps/data_interfaces/views.py [hunk 1]
```diff
@@ -470,6 +475,11 @@ def post(self, request, *args, **kwargs):
             return self.get(request, *args, **kwargs)
         return self.paginate_crud_response
 
+    @use_bootstrap3
+    @use_knockout_js
```
Comment (biyeun) [inline corehq/apps/data_interfaces/views.py]: please pull / rebase latest master into this branch and remove `use_knockout_js` :)

**Comment by pnaw94** [comment]
@biyeun updated

**Comment by gcapalbo** [comment]
@pnaw94 i tested this locally and it looks good, there's just one issue: when i delete a case group, the modal is hidden but the page stays grayed and frozen and i can't click any other buttons or links

**Comment by pnaw94** [comment]
@biyeun @gcapalbo updated

**Comment by gcapalbo** [comment]
@pnaw94 thanks, looks good.  one favor to ask: going forward can we add new commits rather than amending the latest commit for pr feedback?  it just makes it a little difficult to tell what changed otherwise

**Pull request merged**